src/Sql/Sql/help/Update-AzSqlSyncMember.md

```diff
@@ -88,7 +88,7 @@ Type: System.Management.Automation.PSCredential
 Parameter Sets: (All)
 Aliases:
 
-Required: True
+Required: False
 Position: Named
```

Comment:
Note:  please remove any changes to the binary file below (PS-VSPrompt.lnk)

Reply:
done